deploy/20_ocm-agent-operator.Role.yaml

```diff
@@ -4,61 +4,73 @@ metadata:
   name: ocm-agent-operator
   namespace: openshift-ocm-agent-operator
 rules:
-- apiGroups:
-  - ""
-  resources:
-  - services
-  - services/finalizers
-  - configmaps
-  - secrets
-  verbs:
-  - '*'
-- apiGroups:
-  - ""
-  resources:
-  - pods
-  verbs:
-  - get
-  - list
-  - watch
-- apiGroups:
-  - apps
-  resources:
-  - deployments
-  verbs:
-  - '*'
-- apiGroups:
-  - monitoring.coreos.com
-  resources:
-  - servicemonitors
-  verbs:
-  - '*'
-- apiGroups:
-  - apps
-  resources:
-  - deployments
-  - deployments/finalizers
-  - replicasets
-  verbs:
-  - get
-  - list
-  - watch
-  - update
-- apiGroups:
-  - ocmagent.managed.openshift.io
-  - ocmagent.managed.openshift.io/finalizers
-  resources:
-  - '*'
-  verbs:
-  - get
-  - list
-  - watch
-  - update
-  - patch
-- apiGroups:
-  - networking.k8s.io
-  resources:
-  - networkpolicies
-  - networkpolicies/finalizers
-  verbs:
-  - '*'
+  - apiGroups:
```

Comment:
Does the intent matter? It is different with before

Reply:
I am not sure if the previous format without indentation for this `20_ocm-agent-operator.Role.yaml` has any impact but I just make 2 space indentation in `rules` section in this file to align with the pattern of other RBAC file such as `20_ocm-agent-operator.ClusterRole.yaml` and `20_ocm-agent.Role.yaml` and it works all good.